cfgov/jinja2/v1/_includes/molecules/form-field-with-button.html

```diff
@@ -0,0 +1,50 @@
+{# ==========================================================================
+
+   form-field-with-button.render()
+
+   ==========================================================================
+
+   Description:
+
+   Creates a form field with text when given:
+
+   settings:  An object used to customize the markup.
+
+   settings.btn_text:          The text for the submit button.
+
+   settings.field.required:    Boolean indicating wether the field is
```

Comment:
fixed in https://github.com/cfpb/cfgov-refresh/commit/69da2061af1629219bd5b5f70e34a06e49c57ba9